src/engine.rs

```diff
@@ -380,6 +382,26 @@ impl Engine for DevmodeEngine {
     }
 }
 
+struct DisplayBlock<'b>(&'b Block);
```

Comment:
Can we just implement display traits for the structs in the SDK?

Reply:
Either way, the logging would have to change from debug output (i.e. `{:?}`) to display (i.e. `{}`).  An argument could be made that its up to the engine writer on how they want those to be formatted in the log messages.